scenarios/zero_arc_create_and_read/src/main.rs

```diff
@@ -0,0 +1,177 @@
+        ctx.get_mut()
+            .holochain_config_mut()
+            .with_target_arc_factor(0);
+    }
+    start_conductor_and_configure_urls(ctx)?;
+    install_app(ctx, scenario_happ_path!("timed"), &"timed".to_string())?;
+    if ctx.assigned_behaviour() == "zero_read" || ctx.assigned_behaviour() == "zero_write" {
+        try_wait_for_full_arc(ctx)?;
+    }
+    Ok(())
+}
+
+/// Writes timed entries
+fn agent_behaviour_zero_write(
+    ctx: &mut AgentContext<HolochainRunnerContext, HolochainAgentContext<ScenarioValues>>,
+) -> HookResult {
+    let _: ActionHash = call_zome(
+        ctx,
+        "timed",
+        "created_timed_entry",
+        TimedEntry {
+            created_at: Timestamp::now(),
+        },
+    )?;
+
+    ctx.get_mut().scenario_values.sent_actions += 1;
+
+    // Report number of timed entries created
+    let agent_pub_key = ctx.get().cell_id().agent_pubkey().to_string();
+    let metric = ReportMetric::new("zero_arc_create_and_read_entry_created_count")
+        .with_tag("agent", agent_pub_key)
+        .with_tag("arc", "zero")
+        .with_field("value", ctx.get().scenario_values.sent_actions);
+    ctx.runner_context().reporter().clone().add_custom(metric);
+
+    // Report the number of open connections
+    let app_client = ctx.get().app_client();
+    let network_stats = ctx
+        .runner_context()
+        .executor()
+        .execute_in_place(async move { Ok(app_client.dump_network_stats().await?) })?;
+
+    let metric = ReportMetric::new("zero_arc_create_and_read_open_connections")
+        .with_tag("arc", "zero")
+        .with_field("value", network_stats.connections.len() as u32);
+    ctx.runner_context().reporter().clone().add_custom(metric);
+
+    Ok(())
+}
+
+/// Reads the timed entries written by other zero arc nodes.
+fn agent_behaviour_zero_read(
+    ctx: &mut AgentContext<HolochainRunnerContext, HolochainAgentContext<ScenarioValues>>,
+) -> HookResult {
+    let found: Vec<Record> = call_zome(ctx, "timed", "get_timed_entries_network", ())?;
+
+    let found = found
+        .into_iter()
+        .filter(|r| {
+            !ctx.get()
+                .scenario_values
+                .seen_actions
+                .contains(r.action_address())
+        })
+        .collect::<Vec<_>>();
+
+    let reporter_handle = ctx.runner_context().reporter().clone();
+    let agent_pub_key = ctx.get().cell_id().agent_pubkey().to_string();
+    for new_record in &found {
+        let timed_entry: TimedEntry = new_record
+            .entry()
+            .to_app_option()
+            .map_err(|e| anyhow!("Failed to deserialize TimedEntry: {}", e))?
+            .unwrap();
+
+        let metric = ReportMetric::new("zero_arc_create_and_read_sync_lag");
+        let lag_s = (metric
+            .timestamp
+            .clone()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .unwrap()
+            .as_micros()
+            - timed_entry.created_at.as_micros() as u128) as f64
```

Comment:
This makes me realize that clock skew is something we need to take seriously for our wind tunnel runners. Not related to this PR but I'm going to make an issue for it.